golem-worker-service-base/src/api_definition/http/http_api_definition.rs

```diff
@@ -379,27 +379,27 @@ mod tests {
 
     #[test]
     fn expression_with_predicate0() {
-        test_string_expr_parse_and_encode("1>2");
+        test_string_expr_parse_and_encode("${ 1 < 2 }");
     }
 
     #[test]
     fn expression_with_predicate1() {
-        test_string_expr_parse_and_encode("${request.path.user-id > request.path.id}");
+        test_string_expr_parse_and_encode("${ request.path.user-id > request.path.id }");
     }
 
     #[test]
     fn expression_with_predicate2() {
-        test_string_expr_parse_and_encode("${request.path.user-id}>2");
+        test_string_expr_parse_and_encode("${ request.path.user-id > 2 }");
     }
 
     #[test]
     fn expression_with_predicate3() {
-        test_string_expr_parse_and_encode("${request.path.user-id}=2");
+        test_string_expr_parse_and_encode("${ request.path.user-id == 2 }");
     }
 
     #[test]
     fn expression_with_predicate4() {
-        test_string_expr_parse_and_encode("${request.path.user-id}<2");
+        test_string_expr_parse_and_encode("${ request.path.user-id < 2 }");
```

Comment:
Should there be a space now?  or the actual fix is the curly bracce had to go out?

Reply:
curly brace has to go out

otherwise
```
called `Result::unwrap()` on an `Err` value: "Failed to parse: \"${request.path.user-id}<2\""
```